### PR TITLE
[luci-interpreter] Test quantized Mean kernels for ResNet-v1

### PR DESCRIPTION
**Pull request by struss**
Parent Issue: #2730 
Draft PR: #2720

This commit update `Mean` operator test case.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp [hunk 1]
```diff
@@ -92,8 +99,70 @@ TEST(MeanTest, FloatNotKeepDims)
...
+}
+
+TEST(MeanTest, Uint8NotKeepDims)
+{
+  float kQuantizedTolerance = GetTolerance(-1.0, 1.0);
+  std::vector<float> input_data = {0.4, 0.2, 0.3, 0.4, 0.5, 0.6};
+  std::pair<float, int32_t> quant_param = quantizationParams<uint8_t>(-1.0f, 1.0f);
+
+  std::vector<int32_t> axis_data{1};
+  // Tensor input_tensor = makeInputTensor<DataType::FLOAT32>({3, 2}, input_data);
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Mean.test.cpp:145]: Can you remove this code?

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp [hunk 2]
```diff
@@ -92,8 +99,70 @@ TEST(MeanTest, FloatNotKeepDims)
...
+}
+
+TEST(MeanTest, Uint8KeepDims)
+{
+  float kQuantizedTolerance = GetTolerance(-1.0, 1.0);
+  std::vector<float> input_data = {0.4, 0.2, 0.3, 0.4, 0.5, 0.6};
+  std::pair<float, int32_t> quant_param = quantizationParams<uint8_t>(-1.0f, 1.0f);
+
+  std::vector<int32_t> axis_data{1};
+  // Tensor input_tensor = makeInputTensor<DataType::FLOAT32>({3, 2}, input_data);
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Mean.test.cpp:115]: ditto

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp (hunk 1 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/Mean.test.cpp:145]: Okay. I will remove it!

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp [hunk 3]
```diff
@@ -27,6 +27,9 @@ namespace
 
 using namespace testing;
 
+// for quantized Add, the error shouldn't exceed step
+float GetTolerance(int min, int max) { return (max - min) / 255.0; }
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Mean.test.cpp:31]: This is not just for quantized Add (see the comment).

Plus, min and max seem to be float, not int.

If GetTolerance is used in all quantized tests, it had better be placed in "kernels/TestUtils.h". Then, the function can have a more general form, e.g., `float getTolerance(float min, float max, int quantize_steps)`. `quantize_steps` will be 255 for uint8 quantization.

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp (hunk 2 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/Mean.test.cpp]: Okay. I will remove it!

File: compiler/luci-interpreter/src/kernels/Mean.test.cpp (hunk 3 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/Mean.test.cpp:31]: Oh I will fix it!

**Pull request merged**